packages/cardano-services/src/StakePool/TypeormStakePoolProvider/util.ts

```diff
@@ -50,9 +50,13 @@ export const stakePoolSearchSelection = [
 
 export const sortSelectionMap: { [key in SortField]: string } = {
   apy: 'metrics_ros',
+  blocks: 'metrics_minted_blocks',
   cost: 'params.cost',
   lastRos: 'metrics_last_ros',
+  liveStake: 'metrics_live_stake',
+  margin: "(margin->>'numerator')::numeric / (margin->>'denominator')::numeric",
```

Comment:
This might be slow, no changes needed, just something to have in mind in case we need to optimize. Maybe a good idea to add a 'Perf: explanation' comment

Reply:
bcc5e807fb should address this.
It also includes the squash requested in the outer comment.